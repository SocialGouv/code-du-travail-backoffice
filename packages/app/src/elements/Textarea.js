import styled from "@emotion/styled";

const Textarea = styled.textarea`
  -webkit-appearance: none;
  background-color: white;
  border: solid 1px var(--color-border);
  color: inherit;
  flex-grow: 1;
  font-family: inherit;
  font-weight: inherit;
  font-size: inherit;
  line-height: 1;
  padding: 0.4rem 0.6rem 0.575rem;
  resize: none;
`;

export default Textarea;
