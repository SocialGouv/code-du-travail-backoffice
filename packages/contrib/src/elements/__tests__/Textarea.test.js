import React from "react";
import { render } from "@testing-library/react";

import Textarea from "../Textarea";

describe("[Contrib] elements/<Textarea />", () => {
  it("should match snapshot", () => {
    const { container } = render(<Textarea />);

    expect(container).toMatchSnapshot();
  });
});
