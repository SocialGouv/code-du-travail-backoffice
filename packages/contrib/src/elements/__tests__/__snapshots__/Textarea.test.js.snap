// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Contrib] elements/<Textarea /> should match snapshot 1`] = `
<div>
  <textarea
    class="Textarea-sc-1raibsp-0 SZVUP"
  />
</div>
`;
